packages/babel-core/src/transformation/util/clone-deep.ts

```diff
@@ -29,5 +43,14 @@ function deepClone(value: any, cache: Map<any, any>): any {
 
 export default function <T>(value: T): T {
   if (typeof value !== "object") return value;
-  return deepClone(value, new Map());
+
+  if (process.env.BABEL_8_BREAKING) {
+    return deepClone(value, new Map(), false);
+  } else {
+    try {
+      return deepClone(value, new Map(), true);
+    } catch (_) {
+      return structuredClone(value);
```

Comment:
This only works in node 17+, but `v8.serialize` doesn't work in browsers and other runtimes.

Reply:
Can we use `structuredClone` in Babel 8?

Reply:
https://github.com/babel/babel/issues/16967#issuecomment-2486557607
Unfortunately it is slow, less than half the performance of current implementations.🤦‍♂️